core/app/services/spree/remove_line_item.rb

```diff
@@ -0,0 +1,14 @@
+module Spree
+  class RemoveLineItem
+    prepend Spree::ServiceModule::Base
+
+    def call(order:, line_item:, options: nil)
+      options ||= {}
+      ActiveRecord::Base.transaction do
+        line_item.destroy!
+        Spree::AfterAddOrRemove.new.call(order: order, line_item: line_item, options: options)
```

Comment:
`AfterAddOrRemove` is really ambiguous - it should be `AfterAddOrRemoveLineItem`

Reply:
This name should be changed. I just took it from order_contents and looking for better name atm